src/test/shared/clients/defaultIamClient.test.ts

```diff
@@ -0,0 +1,26 @@
+/*!
```

Comment:
These tests should live in `src/test/shared/clients/iamClient.test.ts` , no need for separate files